doc-build/md055.md

```diff
@@ -0,0 +1,20 @@
+This rule is triggered when a table is missing a border. Borders consist of the
+pipe '|' character.
+
+This table is missing a border on the both sides:
```

Comment:
"is missing a trailing pipe on its second line and a leading pipe on its third line"